README.md

```diff
@@ -44,6 +44,24 @@ And finally to build this image you would run
 packer build --var xcode_version=<xcode version> ackee-xcode.pkr.hcl
 ```
 
+#### Publish
+
+To publish the image to the remote so everyone on the team can use it, you have to at first login [Tart][tart] to the remote.
+
+```sh
+tart login ghcr.io
+```
+
+As a username use your Github username and as a password you need to generate new [Personal access token](https://github.com/settings/tokens).
+
+Once you're successfully logged in just run
+
+```sh
+tart push <image> tart push <image> ghcr.io/ackeecz/<image>
```

Comment:
Probably a tag should also be mentioned here 🤔

Reply:
Updated: 64696482ca8128d655bbe79d0ce9e5494d1d7d4a